packages/storybook/src/nimble/anchor-tabs/anchor-tabs-matrix.stories.ts

```diff
@@ -29,9 +32,10 @@ export default metadata;
 // prettier-ignore
 const component = (
     toolbar: TabsToolbarState,
-    [disabledName, disabled]: DisabledState
+    [disabledName, disabled]: DisabledState,
+    widthValue: WidthState
 ): ViewTemplate => html`
-    <${anchorTabsTag} activeid="tab1" style="padding: 15px;">
+    <${anchorTabsTag} activeid="tab1" style="padding: 15px; width: ${() => (widthValue ?? '')};">
```

Comment:
This will render out as invalid / incomplete css, ie: `style="width: "` can we make it render out correctly for either case?